lightning-network-book.tex

```diff
@@ -83,58 +83,58 @@
+    Nodes of the graph are the Lightning Network nodes or participants.
     Edges of the graph are the payment channels.
     The topology of the Lightning Network is publicly broadcasted with the help of the gossip protocol unless nodes decide to act privately.
     This means that the Lightning Network is probably larger than the announced number of nodes.
-    Knowing the Topology is of particular interest for the \gls{source based routing} process of payments in which the sender discovers a route.
-    Also the topology is important for features like the \gls{autopilot}.
+    Knowing the topology is of particular interest for the \gls{source based routing} process of payments in which the sender discovers a route.
+    Also, the topology is important for features like the \gls{autopilot}.
 }}
 \newglossaryentry{transaction}{name={Transaction},
-  description={Transactions are a binary format used by the Bitcoin protocol to transfer Bitcoins from one address to another one.
-    Several transactions build a block which has to be confirmed by the Bitcoin Network through the process of \gls{mining}.
-    Transactions can only be included to a block if they contain a valid signature defined by the input script or previous owner.
-    The first Transaction in each block is called the \gls{coinbase} and generates new Bitcoins.
-    Transactions can also build a \gls{contract} and shall not be confused with \gls{payments}.}}
+  description={Transactions are a binary format used by the Bitcoin protocol to transfer Bitcoins from one address to another.
+    Several transactions are built into a block which has to be confirmed by the Bitcoin network through the process of \gls{mining}.
+    Transactions can only be included in a block if they contain a valid signature defined by the input script or previous owner.
+    The first transaction in each block is called the \gls{coinbase} and generates new Bitcoins.
+    Transactions can also contain \gls{contract}s and should not be confused with \gls{payments}.}}
 \newglossaryentry{source based routing}{name={Source-Based Routing},
   description={
-    On the Lightning Network a sender of the payment decides the route of the Payment.
-    While this decreases the success rate of the routing process it increases the privacy of payments.
-    Due to the SPHINX Mix Format used by the Onion Routing all routing nodes do not no the originator of a payment and the final recipient.
-    Source based routing is fundamentally different to how routing works on the internet protocol.
+    On the Lightning Network the sender of a payment decides the route of the payment.
+    While this decreases the success rate of the routing process, it increases the privacy of payments.
+    Due to the SPHINX Mix Format used by the Onion Routing, all routing nodes do not know the originator of a payment or the final recipient.
+    Source-based routing is fundamentally different to how routing works on the Internet Protocol.
```

Comment:
see for example here you write Internet Protocol instead of internet protocol. so why would it be a payment Hash (or payment hash) instead of a Paymnet Hash?

Reply:
Internet Protocol is a proper noun, as it names the specific thing. In the same way that "Lightning Network" is a proper noun.

Payment hash is generic and doesn't name a specific thing. When you say "Internet Protocol" you are referring to the single Internet Protocol. When you say "payment hash" you could be referring to any payment hash.